### PR TITLE
Changelog 3.5.3

### PR DESCRIPTION
**Pull request by tobiasd**
This should be the last thing merged into the 3.5.3rc branch.

### CODE REVIEW

**Comment by annando** [comment]
Is there something still open or what prevents us from making the release now?

**Comment by tobiasd** [comment]
I think we are good to go, if there are no typos you see in the file

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,64 @@
...
+    Added security and privacy related headers [annando]
+    Removed the external pubsubhubbub service support [annando]
+
+  Friendica Addons:
+    Updates to the translations (ES, ZH-CN) [translation teams]
+    Updates to the documentation [tobiasd]
+    Code revision and refactoring [Hypolite]
+    pumpio, twitter bridges adopted to new background mechanism [annando]
+    Leistungsschutzrecht has a new source list, and a whitelist [annando]
+    retriver marked unsupported due unwanted side-effects [annando]
```
Comment (annando) [inline CHANGELOG]: **retriever**
And: "due **to** unwanted ..."

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,64 @@
...
+    Unicode emoji added [annando]
+    Enhancement to the general content filter [annando]
+    Fixes to buffer, diaspora, libertree, pumpio, gnu social, tumblr, twitter and wppost bridges to redistribute remote_self content [annando]
+    Fixed a bug in securemail settings form [FuzzJunket]
+    external poller addons are deprecated, as this is now a core fuctionality [annando]
```
Comment (annando) [inline CHANGELOG]: **functionality"**

**Pull request merged**